assets/src/scripts/monitoring-location/components/hydrograph/selectors/parameter-data.test.js

```diff
@@ -182,145 +218,52 @@ describe('monitoring-location/components/hydrograph/selectors/parameter-data', (
                 selected: false,
                 timeSeriesCount: 2,
                 periodOfRecord: {
-                    begin_date: '02-01-1980',
-                    end_date: '02-01-2020'
+                    begin_date: '1980-02-01',
+                    end_date: '2020-02-01'
                 }
             }));
             expect(result[2].waterAlert.hasWaterAlert).toBe(false);
 
             expect(result[3]).toEqual(expect.objectContaining({
-                variableID: '52331281',
-                parameterCode: '00010',
-                description:'Temperature C',
-                selected: false,
-                timeSeriesCount: 1,
-                periodOfRecord: {
-                    begin_date: '03-01-1980',
-                    end_date: '03-01-2020'
-                }
-            }));
-            expect(result[3].waterAlert.hasWaterAlert).toBe(true);
-            expect(result[3].waterAlert.subscriptionParameterCode).toEqual('00010');
-
-            expect(result[4]).toEqual(expect.objectContaining({
-                variableID: '52331281F',
-                parameterCode: '00010F',
-                description:'Temperature F',
+                variableID: '52331279',
+                parameterCode: '62611',
+                description:'GW level only 62611',
                 selected: false,
-                timeSeriesCount: 1,
-                periodOfRecord: {
-                    begin_date: '03-01-1980',
-                    end_date: '03-01-2020'
-                }
-            }));
-            expect(result[4].waterAlert.hasWaterAlert).toBe(true);
-            expect(result[4].waterAlert.subscriptionParameterCode).toEqual('00010');
-        });
-
-        it('Returns the groundwater variables if no IV time series are available', () => {
```

Comment:
I removed the test for empty time series and discrete data because they really don't matter. It's what is in variables, uvPeriodOfRecord, and gwPeriodOfRecord.